draft-ietf-tls-esni.md

```diff
@@ -512,6 +512,11 @@ a `ClientECH` with the following values:
 - `enc`, as computed above; and
 - `payload`, as computed above.
 
+If optional configuration identifiers (see {{optional-configs}})) are used, the
+`config_id` field MAY be empty or randomly generated. Unless specified by the
```

Comment:
Let's drop "randomly generated"? I think this would be OK, given how client-facing server behavior is specified. But I don' think we want to put this idea in people's head as a viable option. (Yet.)

Reply:
Oh? {{optional-configs}} has explicitly said it can be random for a while now. (GREASE prescribes a random one too.)